src/protagonist/API/Features/Customer/Requests/UpdateAllImages.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using API.Infrastructure.Requests;
+using DLCS.Core;
+using DLCS.Model;
+using DLCS.Model.Assets;
+using Hydra.Collections;
```

Comment:
Hydra concerns are limited to controller.  Controller receives hydra _stuff_ and converts before passing to mediatr/other classes. Mediatr returns entites/whatever and these are converted before being returned to consumer.